bricks/feature_brick/hooks/pre_gen.dart

```diff
@@ -6,16 +6,23 @@ import 'package:yaml/yaml.dart';
 Future run(HookContext context) async {
   final logger = context.logger;
 
-  final stateManagement =
-      context.vars['state_management'].toString().toLowerCase();
+  final stateManagement = context.vars['state_management'].toString().toLowerCase();
   final isBloc = stateManagement == 'bloc';
   final isCubit = stateManagement == 'cubit';
   final isProvider = stateManagement == 'provider';
   final isRiverpod = stateManagement == 'riverpod';
   final isNone = !isBloc && !isCubit && !isProvider && !isRiverpod;
 
+  bool useFreezed = false;
+  if (isBloc) {
+    useFreezed = context.logger.confirm(
```

Comment:
To save developers from more questions, I am thinking of changing this question into a logger.chooseOne to reduce the amount of questions.

So the questions would contain the values (Equatable, Freezed, None) - This would only have Freezed as an option if the state management is bloc.

Reply:
The wording is a bit tricky on the question, but I think these changes accomplish your idea.

Reply:
Thanks! Will take a look later today!

Reply:
Hey @AdamBuchweitz, Nope! It's all good. I have just been very busy with client work 😅 I will get to this sometime this week!